modules/pom.xml

```diff
@@ -23,6 +23,8 @@
         <aliases.xml.basedir>src/main/aliases</aliases.xml.basedir>
         <hibernate.search.version>${project.version}</hibernate.search.version>
         <hibernate.search.module.slot>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</hibernate.search.module.slot>
+        <infinispan.integration.slot>for-hibernatesearch-${hibernate.search.module.slot}</infinispan.integration.slot>
```

Comment:
Hum, ok. I am wondering why there is a dependency to a specific directory provider in the first place? I would rather expect some automatic discovery of providers present at runtime using some SPI discovery scheme?